cacheops/conf.py

```diff
@@ -38,7 +39,7 @@ def _setup(self):
         except AttributeError:
             raise ImproperlyConfigured('You must specify CACHEOPS_REDIS setting to use cacheops')
 
-        client = (SafeRedis if DEGRADE_ON_FAILURE else redis.StrictRedis)(**redis_conf)
+        client = (SafeRedis if DEGRADE_ON_FAILURE else LocalCachedTransactionRedis)(**redis_conf)
```

Comment:
This line means that `LocalCachedTransactionRedis` is not used if `DEGRADE_ON_FAILURE` is on, which will lead to `redis_client.get()` never use local cache and some your logic failing. Your tests will probably fail now with `DEGRADE_ON_FAILURE = True`.

Anyway, rather than focusing on smaller things like this, it could be better to solve the main problem first - that local cache can't solve transactions. Only queueing invalidations will do. You can however reuse your chainmap and things for that.